src/features/projects/Overview.tsx

```diff
@@ -14,19 +14,325 @@
  * limitations under the License.
  */
 
-import React from 'react';
-
+import React, { useState } from 'react';
+// TODO(chacha912): Extract to CodeBlock component
+import Highlight, { defaultProps } from 'prism-react-renderer';
+import theme, { getStringStyle } from '../documents/prismThemeLight';
 import { useAppSelector } from 'app/hooks';
 import { selectProjectDetail } from './projectsSlice';
 
+type Snippet = 'npm' | 'cdn';
+
 export function Overview() {
   const { project, status } = useAppSelector(selectProjectDetail);
+  const [snippetType, setSnippetType] = useState<Snippet>('npm');
+  const snippet = {
+    npm: `import yorkie from 'yorkie-js-sdk';
+
+async function main() {
+  const client = new yorkie.Client('https://api.yorkie.navercorp.com', {
+    apiKey: '${project?.publicKey}',
+  });
+  await client.activate();
+
+  const doc = new yorkie.Document('my-first-document');
+  await client.attach(doc);
+
+  client.subscribe((event) => {
+    if (event.type === 'peers-changed') {
+      const peers = event.value[doc.getKey()];
+      document.getElementById('peers_count').innerHTML = Object.entries(peers).length;
+    }
+  });
+}
+main();`,
+    cdn: `<!-- include yorkie js -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/yorkie-js-sdk/0.2.17/yorkie-js-sdk.js"></script>
```

Comment:
We can extract the JS SDK address as an environment variable as well. e.g) REACT_APP_JS_SDK_URL